packages/css/src/components/link/README.md

```diff
@@ -13,6 +13,7 @@ Use a link in the following cases:
 - To navigate to another website (see [External links](#external-links))
 - To navigate to an element on the same page
 - To link to emails or phone numbers (start the link with `mailto:` or `tel:`)
+- Wrap 2 or more consecutive buttons and/or links in an [Action Group](?path=/docs/components-buttons-action-group--docs).
```

Comment:
These markdown files are also viewable on github and this link is broken. So links in .md (not the .mdx) should be linked absolute

```suggestion
- Wrap 2 or more consecutive buttons and/or links in an [Action Group](https://designsystem.amsterdam/?path=/docs/components-buttons-action-group--docs).
```

Reply:
PS. Where the link is broken: https://github.com/Amsterdam/design-system/tree/feature/DES-961-dialog-action-group/packages/css/src/components/link